assets/chat/js/messages/ChatUserMessage.js

```diff
@@ -21,8 +21,12 @@ export default class ChatUserMessage extends ChatMessage {
     const attr = {};
 
     if (this.id) attr['data-id'] = this.id;
-    if (this.user && this.user.username)
+    if (this.user && this.user.username) {
+      if (this.user.features && this.user.features.length > 0) {
+        classes.push(this.user.features.join(' '));
+      }
```

Comment:
Appreciate your work! I believe `this.user` is an instance of `ChatUser`, so `features` will always be defined. You can simplify this to
```suggestion
      classes.push(...this.user.features);
```

If you make this change and fix the conflicts, I'll go ahead and merge.